operator/cmd/mesh/profile-dump.go

```diff
@@ -87,8 +86,8 @@ func prependHeader(yml string) (string, error) {
 
 // Convert the generated YAML to pretty JSON.
 func yamlToPrettyJSON(yml string) (string, error) {
-	// YAML objects are not completely compatible with JSON
-	// objects. Let yaml.YAMLToJSON handle the edge cases and
+	// YAML parseObjectSetFromManifest are not completely compatible with JSON
```

Comment:
actually I think this was probably a refactor gone bad?

Reply:
yep.